core/tests/ledger_cleanup.rs

```diff
@@ -192,6 +195,12 @@ mod tests {
     /// - `CLEANUP_SERVICE`: whether to enable the background cleanup service.
     ///   If set to false, the ledger store in the benchmark will be purely relied
     ///   on RocksDB's compaction.  Default: true.
+    ///
+    /// Fifo-compaction settings:
+    /// - `FIFO_COMPACTION`: if true, then RocksDB's Fifo compaction will be
+    ///   used for storing data shreds.  Default: false.
+    /// - `SHRED_DATA_CF_SIZE`: the maximum size of the data-shred column family.
+    ///   Default: 400 * 1024 * 1024 * 1024.
```

Comment:
nit: `Default: DEFAULT_SHRED_DATA_CF_SIZE` to avoid inconsistencies if the constant changes. Right now `DEFAULT_SHRED_DATA_CF_SIZE` is only `125`, so this comment may already be outdated 😃